website/api/controllers/deliver-talk-to-us-form-submission.js

```diff
@@ -46,10 +46,10 @@ module.exports = {
       required: true,
       description: 'What this user will be using Fleet for',
       isIn: [
-        'vulnerability-management',
-        'device-management',
-        'endpoint-ops-it',
-        'endpoint-ops-security',
+        'vm',
+        'dm',
```

Comment:
```suggestion
        'mdm',
```
![image](https://github.com/fleetdm/fleet/assets/108141731/75371678-1652-44b1-91cf-c3e4565cbcc1)